client/client.go

```diff
@@ -41,6 +41,8 @@ type Region struct {
 type Client interface {
 	// GetClusterID gets the cluster ID from PD.
 	GetClusterID(ctx context.Context) uint64
+	// GetMemberInfo gets the member ID from PD
+	GetMemberInfo(ctx context.Context) ([]*pdpb.Member, error)
```

Comment:
I think the comment could be changed as `// GetMemberInfo gets the membes Info from PD`